docs/flow/concepts/configuration_language.md

```diff
@@ -61,6 +61,13 @@ supports more complex expressions, such as:
 
 Expressions may be used for any attribute inside a component definition.
 
+### Referencing component exports
+
+The most common expression is to reference the exports of a component like
+`local.file.password_file.content`. A reference to a component's exports is
+formed by merging the component's name (e.g., `local.file`), label (e.g.,
```

Comment:
should we add `label if exists`?

Reply:
Good point. It feels a little weird to squeeze into the sentence, but I'll add a second sentence for components that don't use labels